src/phantom.js

```diff
@@ -94,6 +95,12 @@ export default class Phantom {
         this.process.stdout.pipe(new Linerstream()).on('data', data => {
             const message = data.toString('utf8');
             if (message[0] === '>') {
+                //Server end has finished NOOP,
+                //lets allow NOOP again..
+                if (message === '>NOOP') {
```

Comment:
Yeah, you could change to a const.  I was trying to make as little MOD's as possible.